truss/templates/server/model_wrapper.py

```diff
@@ -264,10 +264,16 @@ async def __call__(
         async with deferred_semaphore(self._predict_semaphore) as semaphore_manager:
             response = await self.predict(payload, headers)
 
-            processed_response = await self.postprocess(response)
-
             # Streaming cases
             if inspect.isgenerator(response) or inspect.isasyncgen(response):
+                if hasattr(self._model, "postprocess"):
+                    logging.warning(
```

Comment:
open to other ideas on how to handle this scenario -- we could throw an exception if the user has postprocess defined and is trying to do streaming from predict. I figured that it's fairly subtle, and not worth erroring out on. Left a warning instead.

Reply:
i think the warning is fine.